pages/dkp/kommander/2.2/release-notes/2.2.3/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: layout.pug
+navigationTitle: DKP 2.2.3 Release Notes
+title: DKP 2.2.3 Release Notes
+menuWeight: 30
+excerpt: View release-specific information for DKP 2.2.3
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.2.3 was released on October 14th, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download DKP[/button]
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install DKP.</p>
+
+## Release summary
+
+Welcome to D2iQ Kubernetes Platform (DKP) 2.2.3! This release fixes reported issues, integrates changes from previous releases, and maintains compatibility and support for other packages used in Kommander.
+
+DKP 2.2.x supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using DKP 2.2.x must be running a Kubernetes version in this range.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.21.0  |
+| **Maximum**  | 1.22.x  |
+| **Default**  | 1.22.8  |
+
+## Fixes and Improvements
+
+### Chartmuseum not hosting with the cert provided to Kommander during installation, x509 unknown error on Workload cluster (D2IQ-93002)
+
+[Looking at the ticket, do we have a fix for this? We've resolved CA certs below but not the connected missing airgap bundle ticket.]
+
+### Cannot resolve CA certificate when provided as custom certificate (D2IQ-91798)
+
+To resolve this issue, use a custom image for the Kommander federation controllers that includes the patch. The easiest way to do that is by using the installer config. This works both when:
+
+* Installing a new cluster
+
+* Re-running `dkp install kommander` on an existing v2.2.2 cluster
+
+In the installer configuration, under apps, add the following lines:
+```
+  kommander:
+    enabled: true
+    values: |
+      controller:
+        containers:
+          manager:
+            image:
+              repository: mesosphere/kommander2-federation-controller-manager
+              tag: <patched_image_tag>
+```
```

Comment:
Is this really how it would work anymore @cbuto @cwyl02 @gracedo ?
If they download the new DKP CLI for 2.2.3, and then run
`dkp install kommander`
or, upgrade, or whatever it is that they would want, 
wouldn't this come with that?

This _would_ work, but not sure if that's easier than just running install from the new dkp CLI file, etc., because we would have to tell them this tag that's in a private section of Docker Hub, etc.

These are the "workaround" steps, compared to the preferred workflow.

Reply:
@msdolbey you are correct, these are the 'hotfix' instructions.  The 2.2.3 release includes the necessary fix (see above) and so we don't need the the hotfix instructions.  It will just work properly now.

```suggestion
```